relay-server/src/actors/global_config.rs

```diff
@@ -0,0 +1,60 @@
+use std::sync::Arc;
+
+use relay_dynamic_config::GlobalConfig;
+use relay_system::{Addr, Interface, Service};
+use tokio::sync::mpsc;
+
+use crate::actors::project_cache::ProjectCache;
+
+// No messages are accepted for now.
+#[derive(Debug)]
+pub enum GlobalConfigMessage {}
+
+impl Interface for GlobalConfigMessage {}
```

Comment:
You do not have to write this, as there an implementation of the `Interface` for services without messages, see 
https://github.com/getsentry/relay/blob/22a026f4720f964e33144c7fb78b6ba4a6b1bfee/relay-system/src/service.rs#L119-L120
The interface in your service will be just a simple unit `()`